policyengine_us/variables/gov/aca/eligibility/is_medicare_eligible.py

```diff
@@ -0,0 +1,18 @@
+from policyengine_us.model_api import *
+
+
+class is_medicare_eligible(Variable):
+    value_type = bool
+    entity = Person
+    label = "Person is eligible for Medicare"
+    definition_period = YEAR
+    documentation = (
+        "CMS, Original Medicare (Part A and B) Eligibility and Enrollment"
+        "https://www.cms.gov/medicare/enrollment-renewal/health-plans/original-part-a-b"
+        "Above link includes the following text:"
+        "  Part A coverage begins the month the individual turns age 65"
+    )
+
+    def formula(person, period, parameters):
+        MEDICARE_ELIGILITY_AGE = 65
```

Comment:
can you make this a policy parameter? here's a source https://www.law.cornell.edu/uscode/text/42/426#a_1